WordPress/Classes/Services/PostService.m

```diff
@@ -234,6 +234,30 @@ - (void)syncPostsOfType:(PostServiceType)postType
 - (void)uploadPost:(AbstractPost *)post
            success:(void (^)(AbstractPost *post))success
            failure:(void (^)(NSError *error))failure
+{
+    [self uploadPost:post
+forceDraftIfCreating:NO
```

Comment:
`forceDraftIfCreating` seems unaligned here. Is that intentional?

Reply:
I made Xcode align it. I don't think it can align it anymore than that. 😬 The second parameter is longer than the first.

Reply:
It's aligned correctly actually. The alignment is based on the colon.